assets/js/index.js

```diff
@@ -2,10 +2,12 @@ const addTaskIcon = document.getElementById("add-task-icon");
 const addTaskInput = document.getElementById("add-task-input");
 const errorMessage = document.getElementById("error-message");
 const backendErrorMessage = document.getElementById("backend-error-message");
+const todoList = document.getElementById("todo-list");
+const filterItemAll = document.getElementById("filter-item-all");
+const filterItemUncompleted = document.getElementById("filter-item-uncompleted");
+const url = "http://localhost:3030/todos";
```

Comment:
Most of the times we store base url only in global variables. Without the path part. So, Please remove the path from it.

Reply:
Done.